packages/compass-explain-plan/src/components/explain-tree/explain-tree-stage.tsx

```diff
@@ -218,6 +283,7 @@ const ExecutionStats: React.FunctionComponent<ExecutionstatsProps> = ({
                   prevStageExecTimeMS={prevStageExecTimeMS}
                   curStageExecTimeMS={curStageExecTimeMS}
                   totalExecTimeMS={totalExecTimeMS}
+                  formatDisplayTime={milliSecondsToNormalisedValue}
```

Comment:
Can this be changed dynamically for any reason? Doesn't look like this needs to be a prop on a component 🤔

Reply:
Maybe I misinterpreted but I noticed that this component (`Clock`), although not being used anywhere other than this, was built pretty generic. Probably the idea when this was built, was to keep it in re-usable state for future. Hence I tried to keep index tree related logic out of it as much as possible.

Reply:
Feel free to keep it as-is. FWIW it also doesn't make sense to have these three props, width, height, and stroke width as props, they should never be different for this component

Reply:
Yea ok I just re-read what I mentioned and looked again at the `Clock` component. The way this component has been implemented is definitely not re-usable elsewhere as of now because of the function `drawElapsedArc` and other calculation like figuring out exec time for current stage.

I was definitely overthinking this, a commit for simplifying this coming along.